packages/linux-firmware/linux-firmware.spec

```
@@ -0,0 +1,89 @@
+%global debug_package %{nil}
+
+%global _firmwarepath   /usr/lib/firmware
+
+# Many of the firmware files have specialized binary formats that are not supported
+# by the strip binary used in __spec_install_post macro. Work around build failures
+# by skipping striping.
+%global __strip /usr/bin/true
+
+Name:       %{_cross_os}linux-firmware
+Version:    20230625
+Release:    1%{?dist}
+Summary:    Firmware files used by the Linux kernel
```

nit: I try to avoid the column-aligned layouts since those can lead to whitespace-related churn in future diffs, where other lines have to be adjusted to preserve the column

```suggestion
Name: %{_cross_os}linux-firmware
Version: 20230625
Release: 1%{?dist}
Summary: Firmware files used by the Linux kernel
```

---

I was wondering about that when looking at other specs, but muscle memory got the better of me. Fixed.